README.md

```diff
@@ -1,3 +1,4 @@
+![Swift-Logo](https://devimages.apple.com.edgekey.net/assets/elements/icons/swift/swift-128x128.png)
```

Comment:
My preference would be to link to an asset from `swift.org`.  The primary SVG should used as a the website header would probably work.